README.md

```diff
@@ -150,6 +150,14 @@ return [
         'input' => 'bg-gray-100 border border-gray-200 w-full max-w-full p-2 rounded-sm shadow-input text-gray-800 text-base',
         'button' => 'inline-flex place-center px-4 py-3 h-10 border-0 bg-purple-500 hover:bg-purple-600 active:bg-purple-600 overflow-hidden rounded-sm text-white leading-none no-underline',
     ],
+    
+     /**
+     * The positioning of the bubble and the form, change this between rtl and ltr.
+     * If you want to use RTL(Right to Left), you must have your layout set to RTL like this :
+     * <html lang="ar-TN" dir="rtl">
+     * By default, the value of this is LTR
+     */
+    'direction' => 'ltr'
```

Comment:
Could we use full values instead of abbreviations? `left-to-right` instead of `ltr`